components/tickets/PurchaseBottomSheet.tsx

```diff
@@ -10,110 +12,143 @@ import FlexRow from '@/components/shared/FlexRow'
+      priceBpk: priceData?.creditBpkUsed ?? '',
+      priceNpk: priceData?.creditNpkUsed ?? '',
+    }
+
+    const mutation = useMutation({
+      mutationFn: (bodyInner: InitiatePaymentRequestDto) =>
+        clientApi.ticketsControllerInitiateTicketPayment(bodyInner),
+    })
+
+    // eslint-disable-next-line unicorn/consistent-function-scoping
+    const handlePressPay = () => {
+      mutation.mutate(body, {
+        onSuccess: (data) => {
+          console.log('onSuccess', JSON.stringify(data.data, undefined, 2))
+          router.push({
+            pathname: '/purchase/payment',
+            params: { paymentUrl: encodeURI(data.data.paymentUrl ?? '') },
+          })
+        },
+      })
+    }
 
-            {/* Show creditBPKUsed nly if it is defined and "non-zero" */}
-            {!priceData.creditBPKUsed || priceData.creditBPKUsed === 'PT0S' ? null : (
+    // const renderFooter = useCallback(
+    //   (footerProps: BottomSheetFooterProps) => {
+    //     return (
+    //       // eslint-disable-next-line react-native/no-inline-styles
+    //       <BottomSheetFooter style={{ backgroundColor: 'white' }} {...footerProps}>
+    //         <View className="px-5">
+    //           <Link href="/" asChild>
+    //             <Button>{t('pay')}</Button>
+    //           </Link>
+    //         </View>
+    //       </BottomSheetFooter>
+    //     )
+    //   },
+    //   [t],
+    // )
+
+    return (
+      <>
+        <BottomSheet
+          ref={ref}
+          enableDynamicSizing
+          // footerComponent={renderFooter}
+          bottomInset={purchaseButtonContainerHeight}
+          snapPoints={snapPoints}
+        >
+          {priceData ? (
+            <BottomSheetContent cn="g-3" hideSpacer>
               <FlexRow>
-                <Typography variant="default">Stiahnuté z bonusovej karty</Typography>
+                <Typography variant="default">
+                  Parkovanie {formatDuration(durationFromPriceDate ?? 0)}
+                </Typography>
                 <Typography variant="default-bold">
-                  {formatPeriodOfTime(priceData.creditBPKUsed)}
+                  {formatPrice(priceData.priceWithoutDiscount)}
                 </Typography>
               </FlexRow>
-            )}
 
-            {/* Check if it is present (null/undefined, but show if it is 0) */}
-            {priceData.tax == null ? null : (
-              <FlexRow>
-                <Typography variant="default">Dan</Typography>
-                <Typography variant="default-bold">{formatPrice(priceData.tax)}</Typography>
-              </FlexRow>
-            )}
+              {/* Show creditNpkUsed only if it is defined and "non-zero" */}
+              {!priceData.creditNpkUsed || priceData.creditNpkUsed === 'PT0S' ? null : (
+                <FlexRow>
+                  <Typography variant="default">Stiahnuté z návštevníckej karty</Typography>
+                  <Typography variant="default-bold">
```

Comment:
Translations, both npk and bpk

Reply:
Okey, it's time to add translation for this bottom sheet :D